specification/apimanagement/resource-manager/readme.md

```diff
@@ -196,11 +196,21 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2018-01
   - tag: package-2017-03
   - tag: package-2016-10
   - tag: package-2016-07
 ```
 
+### Tag: package-2018-01 and go
+
+These settings apply only when `--tag=package-2018-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2018-01' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/apimanagement/mgmt/2018-01-01/apimanagement

```

Comment:
Note the preview in the path

Reply:
I was thinking about this the other day, what if we made the preview directory a sibling of services?  It would avoid having such a deep directory hierarchy.